.github/workflows/main-branch-deployment.yml

```diff
@@ -0,0 +1,51 @@
+name: Main Branch Deployment
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+      - '[0-9]+-**'
+    paths-ignore:
+      - '**.drawio'
+      - '**.md'
+      - '**.png'
+  pull_request:
+    branches:
+      - main
+    types: [closed]
+
+env:
+  BRANCH_ID: main
```

Comment:
I don't see this env variable used anywhere. Is it necessary?

Reply:
It's not relevant anymore.